### DISCUSSION TITLE
Search/replace tasks taking for so long and constantly "freezing" the computer

### DISCUSSION THREAD

**Discussion by javifcano** [texstudio-org/texstudio#2522] [Q&A]
Hi there. I have been using TexStudio for some many years, and there has never been any problem. However, in the last year or so, the search & replace tasks are taking an annoying amount of time, with the editor freezing for some seconds every time I must do a search/replace operation. It happens even when I only have one file open. I have seen some related discussions in the forums, but nothing that really helps. Is there any way I can fix that? 

Thanks and regards

**Comment by sunderme** [comment]
if there is a problem, open a ticket (issue).
Test case and possibly a screencast to illustrate the issue are necessary.

**Comment by octaeder** [comment]
maybe this has been fixed with https://github.com/texstudio-org/texstudio/issues/2524
